### PR TITLE
Reworked the focalpoint image.

### PR DESCRIPTION
**Pull request by jhavmoeller**
A lot of changes made. Fixed some bugs on the ratio, and added a placeholder-function to set the height of the figure on load.

### CODE REVIEW

File: less/base/base.less [hunk 1]
```diff
@@ -40,11 +40,16 @@ a {
...
 figure {
     margin: 0;
+    background-size: auto 100%;
+    background-position: center center;
+    background-repeat: no-repeat;
 
     img {
         max-width: 100%;
-        width: 100%;
+        // width: 100%;
```
Comment (Dan9boi) [inline less/base/base.less]: why is this necessary?

**Comment by jhavmoeller** [comment]
To prevent stretching images. We already set the width from server, but if the original width off the image is smaller than the figure, it was previously stretched

File: less/base/base.less (hunk 1 above)
Comment (jhavmoeller) [inline less/base/base.less]: To prevent stretching images. We already set the width from server, but if the original width off the image is smaller than the figure, it was previously stretched

**Comment by Dan9boi** [comment]
Great work! :100:

File: less/base/base.less (hunk 1 above)
Comment (hesselberg) [inline less/base/base.less]: Hmm, I think this is an interesting topic to discuss further. Making this change is good for Umbraco based platforms, but considering the novicell frontend package is also used for other platforms, I don't think this is a good decision.

Thoughts, @Dan9boi @jhavmoeller ?

Comment (jhavmoeller) [inline less/base/base.less]: I dont think my changes are more Umbraco-specific than the script was before. The only thing could be the width: 100%, and for my sake we could paste that in again. I just dont see the upside off stretching images per default.

Comment (hesselberg) [inline less/base/base.less]: The 100% width takes into account when we are building things without image resizing tools such as imageresizer.net. Some content editors don't always follow their "training" in regards to uploading images in proper sizes. 

So, when they upload a image that is smaller than it is supposed to be, and we don't have tools like imageresizer.net available, the layout breaks when the image only fills out 2/3 of a container. While stretching images is "bad" for the eye, its even worse when images don't follow their respective container.

I'm all for trying without the width, but that's why it was originally put in :-)

Comment (Dan9boi) [inline less/base/base.less]: I think that we should just put it back, is it wouldn't hurt to have it on an umbraco site, which is using the imageprocessor. @hesselberg and @jhavmoeller.

Can you set it back at make a Mew PR

Comment (Dan9boi) [inline less/base/base.less]: Can you set it back and make a new PR @jhavmoeller, please :)

Comment (mmanovicell) [inline less/base/base.less]: @Dan9boi: Is it an umbraco frontend package? Maybe we should create an cms.umbraco.less file instead if you want cms specific code?

Comment (Dan9boi) [inline less/base/base.less]: **No, it's not an Umbraco frontend package.**
But we can't deny that we, in Novicell, mainly use it for Umbraco projects.
What i'm suggesting is, to make it "less Umbraco specific", and add the "width: 100%" for making it more universal (what @hesselberg said).
That said, it wouldn't hurt an Umbraco site, to have the "width: 100%", because the width is already calculated by the server... :)

So to answer your question, **no, we shouldn't have a CMS specific less file**, at the moment at least @mmanovicell :)
